shared/src/commonMain/kotlin/io/customer/shared/tracking/model/Activity.kt

```diff
@@ -102,7 +102,7 @@ internal sealed interface Activity {
     data class Metric(
         val metricEvent: MetricEvent,
         val deliveryId: String,
-        val deviceToken: String,
+        val deviceToken: String?,
```

Comment:
Reason for this being optional?

Reply:
Same reason as explained [above](https://github.com/customerio/mobile-shared/pull/5#discussion_r1054400603)